frontend/src/components/common/Resource/index.tsx

```diff
@@ -6,3 +6,30 @@ export * from './ResourceListView';
 export { default as ResourceListView } from './ResourceListView';
 export * from './AuthVisible';
 export { default as AuthVisible } from './AuthVisible';
+// adding missing imports based on npm run test
```

Comment:
Please remove this comment, as it makes sense in the case of this PR but we have to think about the code as a whole.